db/migrate/20211109152523_devise_create_users.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+      t.string :username,           null: false
+      t.string :first_name,         null: false
+      t.string :last_name         
+      t.string :role,               null: false, default: "member"
```

Comment:
Nice handling for a default role here - I would still recommend coming back to user roles in a later feature, especially since right now a user role isn't used for anything